ast/dml_test.go

```diff
@@ -139,3 +139,41 @@ func (tc *testDMLSuite) TestLimitRestore(c *C) {
 	RunNodeRestoreTest(c, testCases, "SELECT 1 %s", extractNodeFunc)
 }
 
+func (tc *testDMLSuite) TestWildCardFieldRestore(c *C) {
+	testCases := []NodeRestoreTestCase{
+		{"*", "*"},
+		{"t.*", "`t`.*"},
+		{"testdb.t.*", "`testdb`.`t`.*"},
+	}
+	extractNodeFunc := func(node Node) Node {
+		return node.(*SelectStmt).Fields.Fields[0].WildCard
+	}
+	RunNodeRestoreTest(c, testCases, "SELECT %s", extractNodeFunc)
+}
+
+func (tc *testDMLSuite) TestSelectFieldRestore(c *C) {
+	testCases := []NodeRestoreTestCase{
+		{"*", "*"},
+		{"t.*", "`t`.*"},
+		{"testdb.t.*", "`testdb`.`t`.*"},
+		{"col as a", "`col` AS `a`"},
```

Comment:
Also add a test case for expressions with the "as" e.g. `col + 1 a`.

Reply:
Done.